src/authors/authors.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>Lists of Authors</h1>
+    <div class="wrap">
+         
+     <div class="box">
+            <p>Justyna Rozbicka ...</p>
+            <p class="gitHub">  <a href="#">GitHubLink</a></p>
+        </div>
+        <div class="box">
+         <p>Justyna Rozbicka </p>
+         <p class='gitHub'>  <a href="#">GitHubLink</a></p>
+     </div>
+     <div class="box">
+         <p>Justyna Rozbicka </p>
+         <p class='gitHub'>  <a href="#">GitHubLink</a></p>
+     </div>
+     <div class="box">
+         <p>Justyna Rozbicka </p>
+         <p class='gitHub'>  <a href="#">GitHubLink</a></p>
+     </div>
+     <div class="box">
+         <p>Justyna Rozbicka </p>
+         <p class='gitHub'>  <a href="#">GitHubLink</a></p>
+     </div>
+     <div class="box">
+         <p>Justyna Rozbicka </p>
+         <p class='gitHub'>  <a href="#">GitHubLink</a></p>
+     </div>
+    
+     </div>
+ 
+         <footer>
+     <p class='return'>Redirecting to main page click   <a href="#">Return</a></p>
```

Comment:
Why wrap the link in `<p>`, adding `Redirecting to the main page click` text, and why `return` doesn't target the main app page?